### ISSUE TITLE
Wifi not working after resume from sleep

### ISSUE THREAD

**Issue by LubosKolouch** [banhbaoxamlan/X230-Hackintosh#60]
Hello, thank you for your great setup!

I can't solve one issue though - wifi will not connect after resuming from sleep.

Dmesg paste: https://pastebin.com/J81SnBmi

Any idea what to try?
Thank you...

UPDATE: it stops working even when I disable and enable wifi, not related to sleep.

**Comment by LubosKolouch** [comment]
Works with AirportItlwm-2.0.0-aplha

**Comment by karelo911** [comment]
I tried AirportItlwm 2.0.0 alpha as well as itlwm. I can't seem to get wifi to work again after sleeping or disabling and enabling wifi.

**Comment by karelo911** [comment]
> Works with AirportItlwm-2.0.0-aplha

Can I have a copy of the specific kext you're using to test if it fixes mine? Thanks

**Comment by LubosKolouch** [comment]
This one from the repo:

https://github.com/OpenIntelWireless/itlwm/releases/download/v2.0.0/AirportItlwm_v2.0.0_stable_BigSur.kext.zip

wifi and BT works ok for me... however sleep does not, it wakes instantly with the GLAN error I mentioned in another issue here.

**Comment by karelo911** [comment]
> This one from the repo:
> 
> https://github.com/OpenIntelWireless/itlwm/releases/download/v2.0.0/AirportItlwm_v2.0.0_stable_BigSur.kext.zip
> 
> wifi and BT works ok for me... however sleep does not, it wakes instantly with the GLAN error I mentioned in another issue here.

As much as I want to try this one but I forgot to mention I'm actually on Catalina. Thank you very much for sharing this though.

**Comment by LubosKolouch** [comment]
There's a version for Catalina too

**Comment by karelo911** [comment]
> There's a version for Catalina too

I did try 1.2.0, 1.3.0, 2.0.0 stable and 2.1.0 alpha of itlwm (also tried AirportItlwm earlier but I noticed that itlwm is faster). Wifi connects normally for the first time, but I can't connect anymore (timeout) after turning wifi off either in the menu bar, using the hardware switch, or even using sleep, similar to the behavior you've described, but in Catalina. I tried to ask help from the devs of itlwm/AirportItlwm but I don't have another router to replicate the issue with (I'm using my phone as a hotspot temporarily).
